src/main/java/me/totalfreedom/totalfreedommod/command/Command_op.java

```diff
@@ -54,6 +55,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         FUtil.adminAction(sender.getName(), "Opping " + player.getName(), false);
         player.setOp(true);
+        target.sendMessage(YOU_ARE_OP);
```

Comment:
Same as before, you should wrap this in an if statement to check if the player is not opped before running this command so you do not send this to every player every time you run this command...

Reply:
Pretty sure the NPE will throw since the `target` is going to be nulled and also `player` is nulled.